frontend/src/modules/signup/temp.js

```diff
@@ -0,0 +1,550 @@
+const formDataMapping = [
```

Comment:
Is this `temp.js` file being used anywhere? If not, do we need it in the repo? If yes, can we give it a better name?

Reply:
sorry, i started cloning initiative module and forgot to remove this temp data

Reply:
removed!